### PR TITLE
[DDW-1190] Set decimal places to 2 when showing the sync percentage

### PR DESCRIPTION
**Pull request by danielmain**
This PR follow the suggestion in this issue: https://github.com/input-output-hk/daedalus/issues/3103 to set a fix decimal place to 2.

Expect: 99.89%, 99.9**0**%, 99.91%
Actual: 99.89%, 99.9%, 99.91%

## Screenshots
CleanShot 2023-02-14 at 18 59 40@2x

### Basics

- [ ] PR assigned to the PR author(s)
- [ ] `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa` assigned as PR reviewers
- [ ] All visual regression testing has been reviewed (assign `run Chromatic` label to PR to trigger the run)
- [ ] PR has appropriate labels (`release-vNext`, `feature`/`bug`/`chore`, `WIP`)
- [ ] PR link is added to a Jira ticket, ticket moved to In Review
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR contains screenshots (in case of UI changes)
- [ ] CHANGELOG entry has been added to the top of the appropriate section (_Features_, _Fixes_, _Chores_) and is linked to the correct PR on GitHub
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality

- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with typescript types
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing

- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review

- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/renderer/app/components/loading/syncing-connecting/SyncingProgress/SyncingProgress.tsx [hunk 1]
```diff
@@ -40,6 +42,17 @@ const makePercentageCellStyles = (loaded: boolean) =>
...
   });
 
+const getSafePercentage = (value: number): string => {
+  try {
+    return new BigNumber(value).toFixed(2).toString();
+  } catch (error) {
+    logger.error('SyncingProgress::Percentage::Error parsing sync percentage', {
+      error,
+    });
+    return '-';
```
Comment (tomislavhoracek) [inline source/renderer/app/components/loading/syncing-connecting/SyncingProgress/SyncingProgress.tsx:52]: @danielmain is this `-` a requirement or we can return e.g. `0%` ?

Comment (danielmain) [reply source/renderer/app/components/loading/syncing-connecting/SyncingProgress/SyncingProgress.tsx:52]: I am uncertain if onParsingError we show 0% percentage sync. People will think they have to wait if they see 0% where there is actually an error.

Should we show "Error parsing" ?

Comment (tomislavhoracek) [reply source/renderer/app/components/loading/syncing-connecting/SyncingProgress/SyncingProgress.tsx:52]: hmmm, yeah. Maybe we can keep it as is and show `-`. It is better than showing the error that we are not able to parse the data

**Review by tomislavhoracek — APPROVED** [review]

**Review by dmitrii-gaico — CHANGES_REQUESTED** [review]
The Sync progress now shows the decimal .00 at starting screen but still not fetching the actual progress and always stays integer.
Please see the [record](https://drive.google.com/file/d/1FTqwEHX30I-ba_FxN4O5uLZRYGviuuAq/view?usp=sharing) for details.

File: source/main/utils/handleCheckBlockReplayProgress.ts [hunk 2]
```diff
@@ -68,7 +68,7 @@ const createHandleNewLogLine = (mainWindow: BrowserWindow) => {
       return;
     }
 
-    const progress = Math.floor(parseFloat(unparsedProgress));
+    const progress = parseFloat(unparsedProgress);
```
Comment (tomislavhoracek) [inline source/main/utils/handleCheckBlockReplayProgress.ts:71]: @przemyslaw-wlodek 👍

**Review by tomislavhoracek — APPROVED** [review]

**Pull request merged**